controllers/servingruntime_controller.go

```diff
@@ -437,32 +429,31 @@ func (r *ServingRuntimeReconciler) getRuntimesSupportingPredictor(ctx context.Co
 	}
 
 	restProxyEnabled := r.ConfigProvider.GetConfig().RESTProxy.Enabled
-	srnns := make([]types.NamespacedName, 0, len(runtimes.Items))
+	srNameMap := make(map[string]bool)
```

Comment:
nit: It's actually slightly smaller to make a set with `map[string]struct{}`.. and to insert use `... = struct{}{}`